inc/deployfile.class.php

```diff
@@ -403,6 +403,11 @@ public static function showServerFileTree($rand)
 
         echo "</script>";
 
+        echo '<div class="alert alert-info d-flex" role="alert">';
+        echo '<i class="fas fa-exclamation-circle fa-2x me-2"></i>';
+        echo '<p>' . __('Files need to be uploaded to <kbd>/files/_plugins/glpiinventory/upload/</kbd> folder to be displayed here', 'glpiinventory') . '</p>';
```

Comment:
`<kbd>` is to hint about keyboard input, you should use another HTML tag.

Reply:
![image](https://user-images.githubusercontent.com/7335054/207373242-6d6cf1c1-fba9-46ed-bece-b22dcca7e280.png)